samples/sensor/mcp9808/boards/nucleo_l053r8.overlay

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2019, Linaro Ltd.
```

Comment:
Copyright should likely be adjusted (doesn't seem to be copied?)

Reply:
Fixed.